CatProducer/prod/PAT2CAT_cfg.py

```diff
@@ -64,7 +64,8 @@
     if useMiniAOD:
         process.source.fileNames = ['/store/relval/CMSSW_7_4_6_patch6/RelValTTbar_13/MINIAODSIM/MCRUN2_74_V9-v1/00000/2403409D-1225-E511-B64E-0025905A6132.root']
     ## Hack to run on relval sample
-        process.genMetExtractor.metSource = "slimmedMETs::RECO"
+    # this makes production to fail... need another solution for this
+    #    process.genMetExtractor.metSource = "slimmedMETs::RECO"
```

Comment:
if 'relval' in process.source.fileNames[0]: process.genMetExtractor.metSource = "slimmedMETs::RECO"